src/test/beatmaptest.cpp

```diff
@@ -62,6 +62,12 @@ TEST_F(BeatMapTest, Scale) {
 
     pMap->scale(Beats::THREEFOURTHS);
     EXPECT_DOUBLE_EQ(bpm / 2, pMap->getBpm());
+
+    pMap->scale(Beats::FOURTHIRDS);
```

Comment:
Applied these changes, but I'm not sure why it'll work.
Did the same in beatgridtest.cpp, but I didn't do a testbuild to check it